config/locales/views/en.schedule_rules.yml

```
@@ -3,6 +3,7 @@ en:
     schedule_rules:
       discount: Price Group Discounts
       discount_hint: During the period covered by this schedule rule, overall cost will be discounted by the following percentages.
+      start_time_disabled_help: This instrument can only be reserved on full days
```

I'm not sure about the wording.

Maybe:

```suggestion
      start_time_disabled_help: This instrument can only be reserved for complete days
```

I'm not sure about this one either